src/test/integration/org/apache/cassandra/sidecar/testing/IntegrationTestBase.java

```diff
@@ -158,9 +216,30 @@ protected void createTestKeyspace()
 
     protected void createTestKeyspace(Map<String, Integer> rf)
     {
-        Session session = maybeGetSession();
-        session.execute("CREATE KEYSPACE " + TEST_KEYSPACE +
-                        " WITH REPLICATION = { 'class' : 'NetworkTopologyStrategy', " + generateRfString(rf) + " };");
+        int attempts = 1;
+        ArrayList<Throwable> thrown = new ArrayList<>(5);
+        while (attempts <= 5)
+        {
+            try
+            {
+                Session session = maybeGetSession();
+
+                session.execute("CREATE KEYSPACE IF NOT EXISTS " + TEST_KEYSPACE +
```

Comment:
Because `maybeGetSession` returns `null` in certain caseses where it's not yet connected/the adapter isn't created. Again, may be helped with CASSANDRASC-80?